sympy/stats/crv.py

```diff
@@ -300,6 +300,9 @@ def probability(self, condition, **kwargs):
             rv = [rv for rv in self.values if rv.symbol == domain.symbol][0]
             # Integrate out all other random variables
             pdf = self.compute_density(rv, **kwargs)
+            # return S.Zero if `domain` is empty set
+            if domain.set is S.EmptySet:
```

Comment:
I have used `domain.set` comparison with `S.EmptySet` instead of `domain is S.false` since, that make the code more easy to understand why i am doing that.